tests/test_io_dummy_param.rs

```diff
@@ -29,20 +28,32 @@ mod test {
         let log_q = params.modulus_bits();
         let switched_modulus = Arc::new(BigUint::from(1u32));
         let mut public_circuit = PolyCircuit::new();
+        let num_and_gate = 2;
```

Comment:
I think `test_io_just_mul_enc_and_and_bit` test can always compute `and` for all bits since it requires little computation compared to the other tests, and it is more important to ensure that the most complex pattern for this circuit actually works.